### ISSUE TITLE
Different Alarm messages?

### ISSUE THREAD

**Issue by emiltin** [rsmp-nordic/rsmp_core#64]
Alarm messages come in different types, depending on the value of the aSp field, which can be any of "Issue", "Acknowledge", "Suspend", "Resume", "Request".

Each type require different fields.

Would it be cleaner to simply have different message types, like Alarm, AlarmSuspend, AlarmRequest, etc?

**Comment by otterdahl** [comment]
Yes, perhaps. The aSp (AlarmSpecialisation) field is inherited from the object model of RSMP 1.0 and existed in the old XML schema.
